docs/products/kafka/howto/kafka-tiered-storage-get-started.md

```diff
@@ -1,41 +1,46 @@
 ---
-title: Get started with tiered storage for Aiven for Apache Kafka®
-early: true
+title: Get started with tiered storage
+
 ---
-Aiven for Apache Kafka®'s tiered storage optimizes resources by keeping recent data, typically the most
-accessed, on faster local disks. As data becomes less active, it's transferred to more economical,
-slower storage, balancing performance with cost efficiency.
+import ConsoleLabel from "@site/src/components/ConsoleIcons"
+import {ConsoleIcon} from "@site/src/components/ConsoleIcons"
+
+Aiven for Apache Kafka tiered storage feature optimizes resources by storing recent, frequently accessed data on faster local disks. As data becomes less active, it moves to more economical, slower storage, balancing performance with cost efficiency.
```

Comment:
In this context, "As data becomes less active" is correct because it refers to the general behavior of data within the tiered storage system. Using "As the data" would imply a specific set of data rather than data in general, which is not the intended meaning here.